src/components/hint/index.tsx

```diff
@@ -476,4 +528,4 @@ const styles = StyleSheet.create({
   }
 });
 
-export default Hint;
+export default asBaseComponent<HintProps>(Hint);
```

Comment:
```suggestion
export default asBaseComponent<HintProps, typeof Hint>(Hint);
```
This will solve the error you have in the screen in:
```
position={showBottomHint ? Hint.positions.BOTTOM : Hint.positions.TOP}
```